src/bin/tikv-ctl.rs

```diff
@@ -1082,7 +1083,49 @@ fn main() {
                  .short("f").long("file")
                  .takes_value(true)
                  .help("meta file path"))
-        );
+        )
+        .subcommand(
+            SubCommand::with_name("compact-the-whole-cluster")
+                .about("compact the whole cluster in a specified range in one or more column families")
+                .arg(
+                    Arg::with_name("pd")
+                        .short("p")
+                        .takes_value(true)
+                        .help("pd address, eg. 127.0.0.1:2379"),
+                )
+                .arg(
+                    Arg::with_name("db")
```

Comment:
Please specify the possible values for this argument, like https://github.com/pingcap/tikv/blob/master/src/bin/tikv-server.rs#L300